src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge29.java

```diff
@@ -0,0 +1,77 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+@Component
+@Order(29)
+@RestController
+public class Challenge29 extends Challenge {
+
+    @GetMapping("/localStorageString")
```

Comment:
Let's not mix controller and challenges. Can you please make this endpoint part of the Challengescontroller and map it to `/getSpecialSecret`?

Reply:
And can you add a separate test for this to the controlertests once the code is moved?